things/groups/redis/streams.go

```diff
@@ -32,36 +32,36 @@ func NewEventStoreMiddleware(svc groups.Service, client *redis.Client) groups.Se
 	}
 }
 
-func (es eventStore) CreateGroups(ctx context.Context, token string, groups ...mfgroups.Group) ([]mfgroups.Group, error) {
-	gs, err := es.svc.CreateGroups(ctx, token, groups...)
+func (es eventStore) CreateGroups(ctx context.Context, token string, grps ...mfgroups.Group) ([]mfgroups.Group, error) {
```

Comment:
With the same analogy, we could replace `token` -> `tkn`. For the time being, let's not rename (at least in this PR).